aiohttp/web_runner.py

```diff
@@ -166,15 +167,19 @@ async def start(self) -> None:
         server = self._runner.server
         assert server is not None
         self._server = await loop.create_unix_server(
-            server, self._path, ssl=self._ssl_context, backlog=self._backlog
+            server, str(self._path), ssl=self._ssl_context, backlog=self._backlog
```

Comment:
Test is failing currently, looks like because of this `str()` call. It shouldn't be necessary, the docs say it accepts a Path object since Python 3.7.

Reply:
@Dreamsorcerer if i remove the str call here, i have this error with mypy aiohttp/web_runner.py : 

aiohttp/web_runner.py:170: error: Argument 2 to "create_unix_server" of "AbstractEventLoop" has incompatible type "Union[str, Path]"; expected "Optional[str]"  [arg-type]

using version Python 3.8.10

but i try the commit

Reply:
Fixed: https://github.com/python/typeshed/pull/8376

Just add a `# type: ignore[arg-type]` for now.

Reply:
where i add this in my code ?